e2e/suite.go

```diff
@@ -16,13 +16,15 @@ import (
 	"syscall"
 	"testing"
 
+	singularityinspect "github.com/sylabs/singularity/e2e/inspect"
```

Comment:
this is fine, but why do you need to use a name other than "inspect"?

Reply:
Go complained about it if I didn't have `singularityinspect`, not really sure why...

Reply:
I think I figured out why Go did not like it before, but I'll leave it as it is.